_posts/2017-08-31-foreman-community-newsletter---august-2017.md

```diff
@@ -0,0 +1,124 @@
+users. Please do [log][k_issues] any issues - as always your contributions are
+valuable!
+
+### New Plugin policy
+
+We recently took some time to discuss our policy for "official" plugins - that
+is, plugins which are hosted under the "theforeman" organisation on GitHub. You
+can read the prior discussion [here][plugins_devlist] and the resulting policy
+[here][plugins_website].
+
+One main outcome of this discussion was to ask plugin authors to add a second
+maintainer to the Rubygem, so that the community can help continue maintaining
+the plugin should original authors move on. If your plugin has only one author,
+please consider doing this yourself.
+
+#### Upcoming Events
+
+* [UX Demo - provisioning wizards - 30th August](https://www.youtube.com/watch?v=ZvcAbIuwXsQ)
+
+After we went over through most pages of Foreman and its plugins, we would like
+to concentrate on more complex changes that requires a design or a discussion
+involving all interested parties. We'd like to incrementally discuss progress
+from both current design and implementation perspective as well as generic
+usability questions that pop up. This time we'll focus on various provisioning
+flows, look at first draft and discuss further steps. Demos will be recorded
+and streamed live so anyone from community can participate.
+
+* [Community Demo - 14th September]  (URL to be announced)
+
+Time again for the latest roundup of action from the community, covering core,
+proxies, plugins, and more.  As always, you're welcome to participate in the
+show, either via YouTube Live chat, or via our IRC channel
+(Freenode/#theforeman).
+
+#### Community Demos / Deep Dives
+
+* [Community Demo - 24th August](https://www.youtube.com/watch?v=aOqA8-wpPKQ)
+
+In this demo we covered some minor changes on the code, and then spent a
+significant amount of time on discussing proposals for a new template auditing
+feature from Shim. Your thoughts on this are very welcome in the comments or on
+the mailing list!
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+### New / Updated plugins
+
+#### New plugins:
+
+ATIX.de have released a new plugin aimed at managing snapshots of machines on
+VMware vSphere. THis was actually released in July, but the RPM packaging is
+now done, so you can find it in our plugin repo. For more details, see the
+[announcement][f_snapshot] on the dev mailing list.
+
+In addition, the exisitng
+[foreman_monitoring](https://github.com/theforeman/foreman_monitoring) plugin
+has now been added to the DEB repos as well as the RPM repos - Debian users,
+rejoice!
+
+#### Updated plugins:
+
+- [smart_proxy_dns_infoblox](https://github.com/theforeman/smart_proxy_dns_infoblox) updated to 0.0.6
+- [smart_proxy_omaha](http://github.com/theforeman/smart_proxy_omaha) updated to 0.0.2
+- [smart_proxy_openscap](http://github.com/openscap/smart_proxy_openscap) updated to 0.6.5
```

Comment:
these two should link to the https-URLs, IMHO

Reply:
good catch, ty